components/MatchBanner.js

```diff
@@ -58,15 +68,21 @@ export default class MatchBanner extends React.Component {
 						: <Text>No one has submitted a bid yet, but be patient</Text>
 						}
 
-						<View>
+						<View> {/** Can we style this view, so that these buttons are in a row */}
 							<Button rounded info onPress={() => this.props.close()}>
 								<Text>Close</Text>
 							</Button>
+							{this.state.lotData.driverId
+							?	null
+							:	<Button rounded info onPress={this.handleDelete}>
+									<Text>Delte this Ride</Text>
+								</Button>
+							}
 						</View>
 
 						<TimerCountdown
 							initialSecondsRemaining={pickupTime - now}
-							onTimeElapsed={this.handleFinish}
+							onTimeElapsed={() => {}}
```

Comment:
also here this still needs to get fixed right? maybe with the setTimeout() method

Reply:
What needs to be fixed? I guess that the passenger will want to see something when they are officially matched. But I think that that could just be something that shows up on Matchbanner, maybe? Really, I'm just imagining, it saying like 
"Your driver is 3 minutes away. View his progress on the map." (assuming that you can actually see him approaching) And maybe some more information about him...
Just thinking while I type, but yeah, this probably needs some work. I'll open an issue about it.